src/usr/share/batwatch/batwatch.py

```diff
@@ -53,124 +46,150 @@ def __eq__(self, other):
         return self.__dict__ == other.__dict__
 
     def __ne__(self, other):
-        return not self.__eq__(other)
+        return not self == other
```

Comment:
Why this equals over the other one?

Reply:
Oh, I think I figured it out. They are equivalent, aren't they?

Reply:
Yes they are.